docs/how-to/use-trained-models.txt

```diff
@@ -0,0 +1,197 @@
+
+You may also query the top n checkpoints to receive a sorted list of
+checkpoints using the ``top_n_checkpoints`` method. This method also accepts
+``sort_by`` and ``smaller_is_better`` arguments.
+
+The following snippet returns the top 5 checkpoints for a specified experiment.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoints = Determined.get_experiment(id).top_n_checkpoints(5)
+
+
+The :class:`TrialReference <determined.experimental.TrialReference>` is used for
+more fine grained control over checkpoint selection. It contains a
+``top_checkpoint`` method which mirrors the above ``top_checkpoint`` for an
+experiment. It also contains ``select_checkpoint`` which offers 3 ways to query
+checkpoints.
+
+#. ``best``: returns the best checkpoint based on validation metrics as discussed above. When using ``best``, ``smaller_is_better`` and ``sort_by`` are also accepted.
+#. ``latest``: returns the most recent checkpoint for the trial.
+#. ``uuid``: returns the checkpoint for the trial that matches the specified uuid.
+
+
+The following snippet showcases how to use the different modes for selecting
+checkpoints.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  trial = Determined.get_trial(id)
+
+  best_checkpoint = trial.top_checkpoint()
+
+  most_accurate_checkpoint = trial.select_checkpoint(
+    best=True,
+    sort_by="accuracy",
+    smaller_is_better=False
+  )
+
+  most_recent_checkpoint = trial.select_checkpoint(latest=True)
+
+  specific_checkpoint = trial.select_checkpoint(uuid="uuid-for-checkpoint")
+
+Using the Checkpoint class
+--------------------------
+The :class:`Checkpoint <determined.experimental.Checkpoint>` class can both
+download the checkpoint from persistent storage and load it into memory in a
+python process.
+
+The following snippet downloads a checkpoint from persistent storage to the
+local file system.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoint = Determined().get_experiment(id).top_checkpoint()
+  checkpoint_path = checkpoint.download()
+
+The ``load`` method of the :class:`Checkpoint
+<determined.experimental.Checkpoint>` class downloads the checkpoint if it
+does not exist on locally and then load it into memory in a python
+process as shown in the following snippet.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  checkpoint = Determined.get_experiment(id).top_checkpoint()
+  model = checkpoint.load()
+
+Tensorflow checkpoints are saved in the ``saved_model`` format and are loaded
+as trackable objects (see documentation for `tf.compat.v1.saved_model.load_v2 <https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/saved_model/load_v2>`_ for details).
+
+PyTorch checkpoints are saved via pickle and loaded as ``nn.Module``
+(see `pytorch serialization <https://pytorch.org/docs/stable/notes/serialization.html>`_ for details).
+
+Download Checkpoints via the CLI
```

Comment:
Are we missing docs for `det experiment download <exp_id>`? 
Or is that follow-on work that we decided not to land yet?

Reply:
This hasn't landed yet. I am planning on opening the PR for it today. I will update the docs accordingly in that PR.